src/hotspot/share/gc/z/zPageAllocator.cpp

```diff
@@ -1155,24 +1155,23 @@ void ZPartition::threads_do(ThreadClosure* tc) const {
 }
 
 void ZPartition::print_on(outputStream* st) const {
-  st->print("Partition %u", _numa_id);
+  st->print("Partition %u ", _numa_id);
   st->fill_to(17);
   st->print_cr("used %zuM, capacity %zuM, max capacity %zuM",
                _used / M, _capacity / M, _max_capacity / M);
 
-  streamIndentor indentor(st, 1);
+  StreamAutoIndentor indentor(st, 1);
   print_cache_on(st);
 }
 
 void ZPartition::print_cache_on(outputStream* st) const {
   _cache.print_on(st);
 }
 
-void ZPartition::print_extended_on_error(outputStream* st) const {
+void ZPartition::print_extended_cache_on(outputStream* st) const {
```

Comment:
I would like to suggest that you flip the 'extended' and 'cache' words:
```suggestion
void ZPartition::print_cache_extended_on(outputStream* st) const {
```

So, that we have the structure:
```
print_<what>_on
print_<what>_extended_on
```